### ISSUE TITLE
New rule suggestion: jsx-singleline-no-parens

### ISSUE THREAD

**Issue by dagerikhl** [palantir/tslint-react#206]
A nice rule to have would be to prevent parantheses around singleline JSX. The inverse rule of `jsx-wrap-multiline` if you will.

```jsx
// bad
const component = () => (<div>I should complain, but I don't.</div>);

// good
const component = () => <div>I'm not complaining.</div>;
```

Hope this would be possible to implement, would be a nifty addition!

**Comment by adidahiya** [comment]
Might be useful, but we're not really interested in new formatting rules in this repo right now, as per https://github.com/palantir/tslint/issues/4534. Could happily live in a custom ruleset outside this repo.
